<!DOCTYPE html>
<!--
    Plain-Academic by Vasilios Mavroudis
    Released under the Simplified BSD License/FreeBSD (2-clause) License.
    https://github.com/mavroudisv/plain-academic
-->

<html lang="en">
<head>
  <title>Tibo Duran</title>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.css" />
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.js"></script>
  <link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css' />
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">	
</head>
<body>


<!-- Navigation -->
	<nav class="navbar navbar-inverse navbar-static-top" role="navigation">
	  <div class="container">
		<div class="navbar-header">
		  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
		</div>

		<!-- Collect the nav links, forms, and other content for toggling -->
		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
		  <ul class="nav navbar-nav">
				  <li><a href="index.html">Home</a></li>
				  <li><a href="#teaching">Teaching</a></li>
				  <li><a href="#publications">Publications</a></li> 
				  <li><a href="#committees">Committees</a></li> 
			  	   <li><a href="#projects">Projects</a></li> 
		  </ul>
		</div>
	  </div>
	</nav>
  
  <!-- Page Content -->
    <div class="container">

        <div class="row">

            <!-- Entries Column -->
            <div class="col-md-8"">
                
                <!-- Main Image -->
                <img class="img-responsive" src="profile.png" alt="" /><br />
                
                <div style="margin-top:3%; text-align:justify;">                
					<p>As a highly motivated and ambitious individual, I am excited to embark on the next chapter of my professional journey as I anticipate my <b> graduation in the Spring of 2023</b>. With a broad range of interests spanning the cutting-edge fields of <b>machine learning, data mining, computer vision, bioinformatics, and medical image analysis,</b> I am eager to contribute my skills and expertise to a dynamic and innovative organization. I have honed my abilities and gained valuable experience through a combination of coursework and hands-on projects, positioning me as a well-rounded and adaptable candidate, ready to hit the ground running in a diverse array of roles. I am confident in my ability to make a meaningful impact in any position within these fields and am eagerly looking forward to exploring the various opportunities available to me.</p>
					
					<p>As a distinguished Ph.D. candidate from the esteemed University of Connecticut Computer Science and Engineering Department, I have had the privilege of working under the guidance of renowned researcher, Dr. Sheida Nabavi, in her esteemed lab. Over the past few years, I have devoted myself to the development of cutting-edge deep learning and traditional computer vision methods to analyze a diverse range of medical images, including X-ray images, Computed Tomosynthesis (CT), histological images, and cell images. My research encompasses a broad spectrum of tasks, including classification, segmentation, synthesizing, and reconstructing images, and has led to significant advancements in the fields of cancer diagnosis, disease prediction, and object reconstruction. My contributions to these fields have been substantial, and I am honored to have played a part in advancing the state of the art in medical image analysis. I am eager to continue my research and make more impactful contributions to this field in the future.</p>
					
					<p> <b>I love research and teaching.</b> They are both important in the field of academia. Research allows for the discovery and advancement of new knowledge and understanding in a particular field. Teaching helps to disseminate this knowledge to the next generation of scholars and practitioners. Together, research and teaching form a symbiotic relationship, with research informing teaching and teaching providing a means to apply and expand upon the knowledge gained through research.</p>
                </div>
            </div> 

            <!-- Contact Info on the Sidebar -->
            <div class="col-md-4">
                <div style="font-family: 'Oswald', sans-serif; font-size: 32px;"><b>Tibo Duran</b></div><br />
                <p><b>tibo.duran@uconn.edu</b><br />
                <p>Department of Pharmacutical Sciences<br />
                University of Connecticut<br />
                69 N Eagleville Rd<br />
                Storrs <br />
                United States<br />
                </p>
		<P>
		  <strong>Research Interests:</strong><br/>
		Molecular Dynamcs<br>
		Computer Vision<br>
		Medical Image Analysis<br>
		Bioinformatics<br>
		Machine Learning<br>
		</p>
		<P>
		  <strong>Education</strong><br/>
		Ph.D., University of Connecticut, Expect 2023<br>
		M.S.,University of Dayton, 2015 <br>
		</p>
            </div>
            
            
										
            <!-- Links on the Sidebar -->
            <div class="col-md-4" style="margin-top:2%">
						       <a href="https://www.linkedin.com/in/jun-b-559b1a95/" title="LinkedIn" target="_blank" rel="noopener noreferrer"><i class="fa fa-linkedin-square" style="font-size:48px"></i></a>&nbsp &nbsp &nbsp &nbsp
             <a href="https://scholar.google.com/citations?user=rk9n4QEAAAAJ&hl=en" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar" style="font-size:48px"></i></a>
              
            </div>
            
            <!-- Teaching -->
            <div class="col-md-8">    
                <h2 id="teaching">Teaching</h2><br/>
		    <strong>Lab Teaching</strong><br/>
		   <ul>
                <li class="paper" words=""><a href="#">CSE 1010 Introduction to Computing for Engineers (Spring 2020, Fall 2021)</a> <br>This course serves as an introduction to the fundamental concepts of computing logic, algorithmic thinking, computing processes, programming languages, and computing environments. </li>
                </ul> 
		 <ul>
                <li class="paper" words=""><a href="#">CSE 1729 Introduction to Principles of Programming (Spring 2021)</a> <br>This course is an introduction to computer programming using a structured programming language. The course will cover the fundamental elements of program design and analysis, and students will learn how to use data and functional abstraction to create correct, efficient, and easy-to-understand programs for a variety of common computing problems. </li>
                </ul> 
		   <strong>Teaching Assistant</strong><br/> 
		 <ul>
			  <li class="paper" words=""><a href="#">CSE 4102 Programming Languages (Fall 2020)</a> <br>This course is focused on the study of programming languages, specifically examining their features and programming paradigms. Topics covered in the course include data types, control structures, run-time environments, and semantics.  </li>
		</ul>
		 <ul>
			  <li class="paper" words=""><a href="#">CSE 3300 Computer Networks and Data Communication (Fall 2019, Spring 2022)</a> <br>This course serves as an introduction to the field of computer networks and data communications. The course will cover the fundamental concepts of network types, components, and topology, protocol architecture, routing algorithms, and network performance. </li>
		</ul>
		 		 <ul>
			  <li class="paper" words=""><a href="#">CSE 2102 Introduction to Software Engineering (Spring 2021)</a> <br>This course covers the fundamental concepts of software engineering including the software life cycle and various software development process models. It will introduce students to the different techniques used in software specification, design methodologies, performance analysis, and verification.</li>
		</ul>
				 <ul>
			  <li class="paper" words=""><a href="#">CPS 542 Database Management Systems,  CPS 562 Advance Database Management Systems (Fall 2018, Spring 2019)</a> <br>This course serves as an introduction to database systems, with a focus on modeling and programming databases. Students will learn about E/R modeling, the theoretical concepts of the relational model, and will gain practical experience working with a commercial DBMS. </li>
		</ul>
            </div>
		
            <!-- Publications -->
            <div class="col-md-8">    
                <h2 id="publications">Publications</h2>
                
                <strong>Peer-reviewed</strong><br/>
                <ul>
                <li class="paper" words="Deep learning, Digital breast tomosynthesis, Breast Cancer"><a href="#">Applying deep learning in digital breast tomosynthesis for automatic breast cancer detection: A review</a><br>Jun Bai, Russell Posner, Tianyu Wang, Clifford Yang, Sheida Nabavi</li>
                </ul>
		    
		<ul>
                <li class="paper" words="Deep learning, Prior Mammogram, FFDM"><a href="#">Applying graph convolution neural network in digital breast tomosynthesis for cancer classification</a><br>Jun Bai, Annie Jin, Andre Jin, Tianyu Wang,  Clifford Yang, Sheida Nabavi</li>
                </ul>    
		  <ul>
                <li class="paper" words="Graph Markov neural network, multimodal, COVID"><a href="#">Semi-supervised classification of disease prognosis using CR images with clinical data structured graph</a><br>Jun Bai, Bingjun Li, Sheida Nabavi</li>
                </ul>
		   <ul>
                <li class="paper" words="Un-supervsied, Prior Mammogram, Cancer"><a href="#">Unsupervised Feature Correlation Network for Localizing Breast Cancer Using History of Mammograms</a><br>Jun Bai, Annie Jin, Andre Jin, Madison Adams, Shanglin Zhou, Caiwen Ding, Cliffor Yang,  Clifford Yang, Sheida Nabavi</li>
                </ul>    
		  <ul>
                <li class="paper" words="3D medical image, Model compressiong, Segmentation"><a href="#">Combining multi-view ensemble and surrogate lagrangian relaxation for real-time 3D biomedical image segmentation on the edge</a><br>Shanglin Zhou, Xiaowei Xu, Jun Bai, Mikhail Bragin</li>
                </ul>
		   <ul>
                <li class="paper" words="Freezing rate, protein stability, molecular dynamics simulations"><a href="#">Molecular Dynamics Modeling Based Investigation of the Effect of Freezing Rate on Lysozyme Stability</a><br>Tibo Duran, Bruna Minatovicz, Ryan Bellucci, Jun Bai,  Bodhisattwa Chaudhuri</li>
                </ul>    
		  <ul>
                <li class="paper" words="Protein aggregation, Molecular dynamics, Protein destabilization mechanisms"><a href="#">Molecular dynamics simulation to uncover the mechanisms of protein instability during freezing</a><br>Tibo Duran, Bruna Minatovicz, Jun Bai, Dongkwan Shin, Hossein Mohammadiarani, Bodhisattwa Chaudhuri</li>
                </ul>
		  <ul>
                <li class="paper" words="Single cell RNA sequencing, GCN, Cell classification"><a href="#">Single-cell classification using graph convolutional networks</a><br>Tianyu Wang, Jun Bai, Sheida Nabavi</li>
                </ul> 
		 <ul>
                <li class="paper" words="Deep learning, GCN, Gleason Grade Classification, Weakly Supervised"><a href="#">Weakly-Supervised Deep Learning Model for Prostate Cancer Diagnosis and Gleason Grading of Histopathology Images</a><br>Mohammad Mahdi Behzadi, Mohammad Madani, Hanzhang Wang, Jun Bai, Ankit Bhardwaj, Anna Tarakanova, Harold Yamase, Ga Hie Nam, Sheida Nabavi</li>
                </ul>
                <strong>Software Artical</strong><br/>
                <ul>
                <li class="paper" words="Nucleus microscope image, Computer vision, Image analysis"><a href="#">Brain nucleus microscope image analysis user interface tool</a> <br>The software is designed to perform multiple functions in relation to 3D object of nucleus microscope images. It can display the images, add annotations to them, perform analysis on the images, register them and reconstruct 3D objects from the images. This software is a powerful tool for researchers and scientists who work with microscope images of nuclei, as it allows them to easily view, analyze, and manipulate the images in a variety of ways. The ability to register and reconstruct 3D objects from the images can also provide valuable insights and information that can aid in scientific research and discovery.</li>
                </ul>
	           <ul>
                <li class="paper" words="Clinical model evaluation, Computer vision, Cancer detection"><a href="#">Prediction of cancer probability using prior and current year images: A user interface</a> <br>The software is developed to assess the capability of a novel deep learning model in predicting clinical trials and compare its performance with radiologists. It will provide a user interface to evaluate the prediction ability of the deep learning model and compare it with the results of radiologists. This software could be useful for researchers and medical practitioners who are working in the field of medical imaging and want to evaluate the accuracy and reliability of deep learning models in predicting clinical outcomes.</li>
                </ul>
		    
		                    <strong>On-going paper</strong><br/>
                <ul>
                <li class="paper" words="Continuous learning, Clinical usage, Deep learning"><a href="#">A continuous learning framework for updating model weights with expert radiologist guidance for improved diagnostic accuracy</a> In process</li>
                </ul>
		    <ul>
                <li class="paper" words="Cancer localization, FFDM, Deep learning"><a href="#">Un-supversied feature correlation model to predict breast abnormal variation map</a> In process</li>
                </ul>
		    
		  <ul>
                <li class="paper" words="Image registration, Computer vision, Review"><a href="#">A review of current status and challenges of  microscope image registration</a> In process</li>
                </ul>
            </div>
	 
	   <div class="col-md-12">    
              <h2 id="committees">Committes</h2>
	    <ul style="list-style: none;">
 		<li>PC member &nbsp&nbsp 2022 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</li>
 		<li>PC member &nbsp&nbsp 2023 ACM Special Interest Group (SIG) on Knowledge Discovery and Data Mining (KDD)</li>
		<li>Reviewer &nbsp&nbsp Medical Physics Journal</li>
		 <li>Reviewer &nbsp&nbsp Current Bioinformatics Journal</li>
		 <li>Reviewer &nbsp&nbsp 2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</li>
	    </ul>
	   </div>
	
       
	
	
	   <div class="col-md-4" >
				<h2 id="projects">Selected Projects</h2>
	     <br><br><a href = "img/firstPaper.png">   
             <img alt="img" src="img/firstPaper.png" class="img-responsive" width="250" height="250"></a>		
           </div>
	   <div class="col-md-5" align="justify"> 
		    <h2></h2>
	    <p> <br><br><strong>Ultraplex Microscopy image reconstruction App development</strong><br />
		The challenge of unraveling the mysterious interplay between individual cells and intricate neural circuits is a daunting task, owing to its overwhelming complexity. However, the current project endeavors to surmount this challenge by creating a state-of-the-art software tool that is capable of seamlessly processing, annotating, registering, analyzing, and reconstructing 3D cells and neural systems with unparalleled precision. This revolutionary tool will serve as the cornerstone for the application of deep learning models and will pave the way for a deeper understanding of the intricacies of the biological world.				
	    </p>				
	   </div>
	   <div class="col-md-4" > 
	     <br><br><br><br><br><br><br><br><a href = "img/secondPaper.png">   
             <img alt="img" src="img/secondPaper.png" class="img-responsive" width="250" height="250"></a>		
           </div>
	   <div class="col-md-5" align="justify"> 
	    <p>  <br><br><strong>Digital Breast Tomosynthesis cancer prediction and localization using Graph Neural Network</strong><br />
		The advent of digital breast tomosynthesis has transformed the landscape of breast imaging diagnosis, thrusting it to unprecedented levels of precision and accuracy. However, despite these remarkable advancements, the automatic detection of breast cancer using the high-resolution, high-volume, and complex images generated by digital breast tomosynthesis has remained a formidable challenge. To conquer this challenge, our research team has embarked on a pioneering journey of developing a revolutionary model for the detection of cancerous 3D mammogram images. In this groundbreaking study, we have developed an innovative model for more precise detection of cancerous 3D mammogram images. This novel model firstly represents 3D mammograms as graphs, then uses a self-attention graph convolutional neural network model to effectively and efficiently learn the features of 3D mammograms, and finally, by utilizing the extracted features, identifies the cancerous 3D mammograms with unparalleled accuracy. This proposed model will undoubtedly pave the way for more efficient and effective breast cancer diagnosis, and a major step forward in the fight against this devastating disease.				
	    </p>				
	   </div>
	 <div class="col-md-4" > 
	     <br><br><br><br><br><br><a href = "img/Third.jpg">   
             <img alt="img" src="img/Third.jpg" class="img-responsive" width="250" height="250"></a>		
           </div>
	   <div class="col-md-5" align="justify"> 
	    <p>  <br><br><strong>Semi-supervised classification of disease prognosis using CR images with clinical data structured graph</strong><br />
		The rapid expansion of global connectivity and urbanization has created a perfect storm for the spread of disease on a worldwide scale. The outbreak of the SARS-COV-2 pandemic has placed immense strain on healthcare systems, particularly in the area of intensive care units. In light of this dire situation, the accurate prognosis of patients' need for intensive care units at the earliest stage of hospital admission has become a top priority, in order to ensure efficient allocation of resources. To this end, we have proposed a groundbreaking clinical data structured graph Markov neural network embedding with computed radiography exam features (CGMNN) to predict the intensive care units demand for COVID-19 patients. This innovative approach utilizes patient chest radiography and clinical data collected at the early stages of hospitalization to make more accurate diagnoses, ensuring that patients receive the care they need in a timely and efficient manner.				
	    </p>				
	   </div>
     <div class="col-md-4" > 
	     <br><br><br><br><br><br><a href = "img/four.jpg">   
             <img alt="img" src="img/four.jpg" class="img-responsive" width="250" height="250"></a>		
           </div>
	   <div class="col-md-5" align="justify"> 
	    <p>  <br><br><strong>Weakly-Supervised Deep Learning Model for Prostate Cancer Diagnosis and Gleason Grading of Histopathology Images</strong><br />
		Prostate cancer is a global scourge, afflicting men worldwide and ranking as the second leading cause of cancer death in the United States. One of the key prognostic features in prostate cancer is the Gleason grading of histopathology images. This grading system, which is based on the tumor architecture of Hematoxylin and Eosin (H&E) stained whole slide images (WSI) is assigned by pathologists and is known to be both time-consuming and prone to inter-observer variability. In recent years, deep learning algorithms have been employed to analyze histopathology images, with promising results in the grading of prostate cancer. However, these algorithms rely on fully annotated datasets, which are both costly and time-consuming to generate. In this work, we propose a novel weakly-supervised algorithm to classify prostate cancer grades. The proposed algorithm is composed of three steps: (1) extracting discriminative areas in a histopathology image by utilizing the Multiple Instance Learning (MIL) algorithm based on Transformers, (2) representing the image by constructing a graph using the discriminative patches, and (3) classifying the image into its Gleason grades by developing a Graph Convolutional Neural Network (GCN) based on the gated attention mechanism. This innovative algorithm is set to revolutionize the field of prostate cancer diagnosis and pave the way for more accurate, efficient, and cost-effective treatment of this devastating disease.<br/>				
	    </p>				
	   </div>
           <div class="col-md-4" > 
	      
             <br><br><video width="250px" height="250px" autoplay loop muted>
         
        	<source src="img/Media1.mp4" type="video/mp4"></video>
    	
           </div>
	   <div class="col-md-5" align="justify"> 
	    <p>  <br><br><strong>Molecular dynamics simulation to uncover the mechanisms of protein instability during freezing</strong><br />
		Freezing is a widely used technique in the pharmaceutical industry for the preservation and transportation of biotherapeutics. In order to gain a deeper understanding of the mechanisms and molecular details of protein destabilization upon freezing, multi-scale molecular dynamics simulations of Lactate dehydrogenase (LDH) protein in phosphate buffer with/without ice formation were performed. The simulations were conducted under both fast and slow ice-growing conditions at 243 K, from one or two sides of the simulation box, respectively. The results showed that the rate of ice formation at all-atom simulations was crucial to the stability of LDH, as faster freezing rates resulted in enhanced structural stability and a higher number of intramolecular hydrogen bonds, less flexible protein residues, lower solvent accessibility, and greater structural compactness. Additionally, protein aggregation was investigated through coarse-grained simulations and was found to be initiated by extended protein structures and retained by electrostatic interactions of salt bridges between charged residues and hydrogen bonds between polar residues of the protein. Lastly, the study of the free energy of dissociation through steered molecular dynamics simulation revealed that LDH was destabilized by the solvation of the hydrophobic core and the loss of hydrophobic interactions. This study, for the first time, provides experimentally validated molecular simulations that reveal the detailed mechanisms of LDH destabilization upon ice formation and cryoconcentration of solutes, and it will help to improve the preservation and transportation of biotherapeutics.				
	    </p>				
	   </div>
	   <div class="col-md-4" > 
	      
             <video width="250px" height="250px" autoplay loop muted>
         
        	<source src="img/Media2.mp4" type="video/mp4"></video>    	
           </div>
           <div class="col-md-5" align="justify">
	   </div>
	  <div class="col-md-4" > 
	      
             <video width="250px" height="250px" autoplay loop muted>
         
        	<source src="img/Media3.mp4" type="video/mp4"></video>   	
           </div>
        </div>

    </div>
    <!-- /.container -->
    
    <!-- Other people may like it too! -->
    <a style="color:#b5bec9;font-size:0.8em; float:right;" href="https://github.com/mavroudisv/plain-academic">Plain Academic</a> 
    
</body>

</html>
